lib/estella/parser.rb

```diff
@@ -18,7 +18,8 @@ def field(name, opts = {})
       using = opts[:using] || name
       analysis = opts[:analysis] & @model.default_analysis_fields.keys
       opts[:fields] ||= Hash[analysis.zip(@model.default_analysis_fields.values_at(*analysis))] if analysis
-
+      include_raw = opts.delete(:include_raw)
+      opts[:fields][:raw] = { type: 'keyword' } if include_raw
```

Comment:
this looks good but do we have a use case for it? ie was it prompted by a test failure? sounds like a good option to include regardless just wondering.

Reply:
Sorry didn't see this comment!

Yes- see https://github.com/artsy/gravity/blob/3c060224c56e10c84b26727e0f44fa161868a3c3/app/models/domain/sale.rb#L127

We use estella for the mapping, and that `name` field is desired to be analyzed (for text search), but also stored as a keyword type for filtering/sorting.

Without using estella, I was able to accomplish this via: https://github.com/artsy/gravity/blob/3c060224c56e10c84b26727e0f44fa161868a3c3/app/models/search/indexes/artwork_filter_index.rb#L28-L29

So I _think_ we should keep this and need it (unless I'm missing something with this multi-field storing a string and also a keyword). If you have a field where you want to do both text search but also filtering/sorting, one would use this option.